src/classes/C501_CTRL_ClassReg_CreateAcct.cls

```diff
@@ -0,0 +1,102 @@
+        loginID	= null;
+        Cookie aId = ApexPages.currentPage().getCookies().get('LoginIdClassRegSite');
+        if (aId != null){
+            loginID 	= aId.getValue();
+            loggedIn 	= C501_UTIL_ClassRegUtilities.confirmLoggedIn(loginID,false);
+            pageStatus	= 'new';
+        } else {
+            String	attID 		= ApexPages.currentPage().getParameters().get('a');
+            String 	remember 	= ApexPages.currentPage().getParameters().get('r');
+            loggedIn 			= C501_UTIL_ClassRegUtilities.confirmLoggedIn(attID,true);
+            if(loggedIn){
+            	Integer remberInt = 86400;
+                if(remember != null && remember == '1'){
+                    remberInt = -1;
+                }
+                aId = new cookie('LoginIdClassRegSite', attID, null, remberInt, false);
+                ApexPages.currentPage().setCookies(new Cookie[] {aId});
+                pageStatus	= 'new';
+            } else {
+                pageStatus	= 'fail';
+            }
+        }
+        
+        parent = new Contact();
+        String 	confirmMeth 	= ApexPages.currentPage().getParameters().get('c');
+        
+        family = new Account(
+            RecordTypeId 	= [select id from Recordtype where Name='Household Account'].Id,
+            Name			= 'Website Generated Account'
+        );
+        if(confirmMeth != null && confirmMeth.contains('@')){
+            parent.Email 		= confirmMeth;
+        } else {
+            parent.MobilePhone 	= confirmMeth;
+        }
+    }
+    
+    /*public PageReference confirmLogin(){
+        PageReference nextPage = C501_UTIL_ClassRegUtilities.confirmLogin(confirmation,acctID,remember);
+        if(nextPage == null){
+            pageStatus = 'confirm-fail';
+            return null;
+        } else {
+            // Verified login, so update the Account with details provided.
+            insert family;
+            insert parent;
+            //C501_UTIL_ClassRegUtilities.
+            
+            return nextPage;
+        }
+    }*/
+    
+    public Pagereference goBack(){
+        return C501_UTIL_ClassRegUtilities.generatePageLink('C501_ClassReg_Login',null,null);
+    }
+    
+    public Pagereference createAccount(){
+        
+        /*family.C501_Class_Site_Mobile_Number__c = C501_UTIL_ClassRegUtilities.cleanInputPhone(family.C501_Class_Site_Mobile_Number__c);
+        
+        // next, we need to check if there is already an account with that phone number. If it is them, send a confirmation text and pair them with that account
+        acctID = C501_UTIL_ClassRegUtilities.lookupAccountByPhone(family.C501_Class_Site_Mobile_Number__c);
+        
+        if(acctID != null ){
+            // send the sms
+            C501_UTIL_ClassRegUtilities.sendSMS(acctID,family.C501_Class_Site_Mobile_Number__c);
+            //family.Id = acctID;
+            parent.AccountId = acctID;
+        	// present user with a code screen
+        	pageStatus	= 'confirm';
```

Comment:
Text isn't aligning up - can see there are tabs in the files.  Tabs can mess up display since different viewers render tabs differently.  Can you set your editor to automatically put 4 spaces in for a tab key?  I thought that was default in Visual Studio Code but maybe I changed that for my source files.

Reply:
It lines up in Code and in Developer Console. I will see if I can set this to use 4 spaces instead of tabs. 

I have noticed that it can look screwed up in github and just fine everywhere else.